src/main/resources/application.yml

```diff
@@ -8,8 +8,8 @@ rabbit:
 
 logging.level.root: INFO
 
-wes:
-  url: http://localhost:9090/runs
+rdpc:
+  url: https://api.rdpc-dev.cancercollaboratory.org/graphql
```

Comment:
do we want rdpc dev as default or leave localhost to be overwritten by dev?

Reply:
Might as well leave it at localhost, will change.